STACpopulator/extensions/file.py

```diff
@@ -0,0 +1,65 @@
+"""FileHelper module."""
+
+import functools
+from typing import Dict, TypeVar
+
+import pystac
+import requests
+from pystac.extensions.file import FileExtension
+
+from STACpopulator.extensions.base import ExtensionHelper
+
+# Constants
+T = TypeVar("T", pystac.Asset, pystac.Link)
+HTTP_SERVER_ASSET_KEY = "HTTPServer"
+OPEN_DAP_ASSET_KEY = "OpenDAP"
```

Comment:
I think those should be attributes of the helper provided in `__init__`. These values can be the defaults, but they are technically editable. Notably, the HTTP server case could be reused with other access methods than THREDDS.